Automattic/jetpack
# Boost: add tracking events to ISA report

Record events when the "edit page" and "fix on page" buttons on the ISA report are clicked.

For more, see #31741

## Proposed changes:
* Add a new function called recordBoostEventAndRedirect to record an event and redirect to a new URL.
* Use that function on the two previously mentioned buttons.

### Other information:


## Jetpack product discussion

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Apply patch
* You might want to add a console.log to `recordBoostEventAndRedirect` to check that it's working.
* View the ISA report and click on the edit and fix buttons on any image.
